msteams-platform/concepts/build-and-test/apps-package.md

```diff
@@ -56,11 +56,13 @@ Teams automatically crops your icon to display a square with rounded corners in
 
 An outline icon displays in two scenarios:
 
-* When your app is in use and “hoisted” on the app bar on the left side of Teams.
+* When your app is in use and pinned to the app bar on the left side of Teams.
 * When a user pins your app's message extension.
 
 Ensure the icon is 32x32 pixels. It should be either white with a transparent background or transparent with a white background. No other colors are allowed. The outline icon mustn't contain any additional padding around the symbol.
 
+In Outlook and the Microsoft 365 app, the color icon is used when an application is pinned, which may result in inconsistent appearances across experiences. To maintain the appearance of your app when it is pinned in other experiences, you can specify an optional 32x32 color icon with a transparent background.
```

Comment:
Looks good. Suggested tweaks:

Although the outline icon is used when your app is pinned in Teams, when your app is pinned in Outlook or the Microsoft 365 app, the (scaled down) *color* icon is used instead. This could result in inconsistent appearances across experiences. To ensure your full color app icon looks good when pinned in Outlook and Microsoft 365 app, you can specify an optional 32x32 color icon with a transparent background.